src/pages/InterviewSchedulingPage/components/InterviewScheduler.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import ScheduleSelector from 'react-schedule-selector';
+
+interface InterviewSchedulerProps {
+  numDays?: number;
+  minTime?: number;
+  maxTime?: number;
+  startDate?: Date;
+  reset?: boolean;
+  setDirtyCallback?: () => void;
+  selectedScheduleCallback?: (arg: Date[]) => void;
+}
+
+function InterviewScheduler({
+  numDays = 5,
+  minTime = 11,
+  maxTime = 21,
```

Comment:
nit: what do you think about default maxTime 19?
also probably startHour/endHour is more accurate naming

Reply:
19 is fine. Really depends on how late into the day we want to conduct interviews

yeah startHour/endHour is better, I had minTime/maxTime because that was what react-schedule-selector had